algorithms/schema-free/kmeans_sub.gsql

```diff
@@ -0,0 +1,129 @@
+CREATE QUERY kMeans_sub(int k, float maxChange = 1.0, vertex input, string e_type, bool random = FALSE, BOOL print_accum = TRUE, STRING result_attr="", STRING                    file_path="") returns (ListAccum<Float>) {
+  TypeDef Tuple<int center, float value> distanceTuple;
+  HeapAccum<distanceTuple> (1,value asc) @distanceHeap;
+  ArrayAccum<AvgAccum> @@Array[100][100];
```

Comment:
What do these fixed size arrays do? What do the limits (100) mean?  I know we can't parameterize arrays, so we should have inline comments to explain. Users who need to modify these values need to understand these better.

Reply:
fixed array sizes are for the centroids of each cluster. In the updated commit, there are comments explaining these need to be changed